src/commands/CommandHandler.ts

```diff
@@ -133,6 +135,7 @@ export async function handleCommand(roomId: string, event: { content: { body: st
                 "!mjolnir redact <event permalink>                                   - Redacts a message by permalink\n" +
                 "!mjolnir kick <user ID> [room alias/ID] [reason]                    - Kicks a user in a particular room or all protected rooms\n" +
                 "!mjolnir rules                                                      - Lists the rules currently in use by Mjolnir\n" +
+                "!mjolnir rules matching <entity>                                    - Lists the rules in use that will match this entity e.g. @foo:example.com will show all the user and server rules, including globs, that match them." +
```

Comment:
Nit: instead of just `@foo:example.com`, I'd write `!mjolnir rules matching @foo:example.com`.

Also, could you document whether it works with room ids? Server names? Room aliases?